### ISSUE TITLE
[Feature Request] Default Root Folder

### ISSUE THREAD

**Issue by Pantar2205** [toddrob99/searcharr#26]
Consider allowing an optional setting in settings.py to set a default root folder and therefore not ask users what folder they want to use. Alternatively hide root folders x,y,z from searcharr.

**Comment by toddrob99** [comment]
Considering for v2

**Comment by toddrob99** [comment]
This will be added in v2, and it is already included in v2.0-b3 which is available in the beta branch here as well as the toddrob/searcharr:beta docker image.

Add the following to settings.py and fill in the root folder(s) you want to allow users to choose from. If you want to force one root folder, put only that one in the list.

`sonarr_series_paths = []  # e.g. ["/tv", "/anime"] - can be full path or id value - leave empty to enable all`
`radarr_movie_paths = []  # e.g. ["/movies", "/other-movies"] - can be full path or id value - leave empty to enable all`

You can also use the id values from the Sonarr/Radarr API, e.g. `sonarr_series_paths = [1, 2]`.

**Comment by Pantar2205** [comment]
Thank you.

I added my movie path and it didn't work. I checked the logs and had not formatted the path correct "/a/working/path". So corrected that and thats working beautifully. User sees no questions about quality or paths.

Doing the exact same for series gives me no error log about a bad syntax or bad path. Seems to work, but still asks user from a selection of paths.

for reference, the radarr path setting in settings.py

`radarr_movie_paths = "/media/movies-remove"`

and sonarr

`sonarr_series_paths = "/tv/shows-remove"`

I can comform via bash command pwd that these are the correct paths and also confirm they are correct in both sonarr and radarr.

the quality selection is perfect for both.

**Comment by toddrob99** [comment]
@Pantar2205 can you please post your logs? You should have a log entry that starts with: `No valid Sonarr root folder(s) provided! Using all of the root folders I found in Sonarr:`. I'm interested to see that log entry as well as a few lines before and after.

If you haven't already, please [enable debug logging](https://github.com/toddrob99/searcharr/wiki/Enable-Debug-Logging).

**Comment by Pantar2205** [comment]
Sorry I went digging around myself and It has been resolved but I am not sure how. I noticed I had mistakenly set the container to run as root by mistake. changing that, then setting radarr and sonarr tags to False has fixed the problem without touching the  sonarr path i have set at all. i don't have logs as i deleted them and chowned the config directories when i change my docker compose file. from a settings.py point of view, the only thing i altered was `sonarr_tag_with_username = False` and `radarr_tag_with_username = False`
radarr worked before i did that, but doing that seemed to fix sonarr
